frontends/web/src/routes/settings/bb02-settings.tsx

```diff
@@ -193,9 +194,19 @@ const Content = ({ deviceID }: TProps) => {
         }
         <GoToStartupSettings deviceID={deviceID} />
         <FactoryResetSetting deviceID={deviceID} />
+        {
+          versionInfo ? (
+            <ChangeDevicePasswordSetting
+              deviceID={deviceID}
+              canChangePassword={versionInfo.canChangePassword}
+            />
+          ) : (
+            <StyledSkeleton />
+          )
+        }
       </div>
     </>
   );
 };
 
-export { BB02Settings };
+export { BB02Settings };
```

Comment:
Accidental change? Would be useful to configure your editor to add trailing newlines (or at least not to delete them).